<launch>
	<!-- Robot controller parameters -->
  <node pkg="pioneer_test" type="pioneer3dx_pid_controller" name="pioneer_test">
    <rosparam file="$(find pioneer_test)/pioneer_controller_params.yaml" command="load"/>
    <!--remap from="/RosAria/cmd_vel" to="controller_cmd_vel"/-->
  </node>
  <node pkg="pioneer_test" type="turn_robot_around.py" name="turn_robot_around">
    <remap from="recovery_cmd_vel" to="/RosAria/cmd_vel"/>
  </node>
</launch>
